dashboard/src/components/Header/HeaderLink.tsx

```diff
@@ -18,8 +20,10 @@ class HeaderLink extends React.Component<IHeaderLinkProps> {
   };
 
   public renderInternalLink() {
-    const { currentNamespace, namespaced, to } = this.props;
-    const link = currentNamespace && namespaced ? `/ns/${currentNamespace}/${to}` : to;
+    const { currentCluster, currentNamespace, clustered, to } = this.props;
+
+    let link = currentNamespace ? `/ns/${currentNamespace}/${to}` : to;
+    link = clustered ? `/c/${currentCluster}${link}` : link;
```

Comment:
let's move all the links to `shared/url` so we have a single place with all the links. We can have a section `header` there with these links.

Reply:
that's fine, thanks